sources/packages/api/src/services/notification/gc-notify.model.ts

```diff
@@ -0,0 +1,35 @@
+export interface GCNotifyResult {
+  content: ContentPayload;
+  id: string;
+  reference: string;
+  scheduled_for: string;
+  template: Template;
+  uri: string;
+}
+
+export interface Template {
+  id: string;
+  uri: string;
+  version: number;
+}
+
+export interface ContentPayload {
+  body: string;
+  from_email: string;
```

Comment:
We can add a comment before declaration of interface mentioning why camel case not followed here(as gc notify expects that way.)